go/vt/topo/keyspace.go

```diff
@@ -54,6 +55,20 @@ func (ki *KeyspaceInfo) SetKeyspaceName(name string) {
 	ki.keyspace = name
 }
 
+var invalidKeyspaceNameChars = "/"
+
+// ValidateKeyspaceName checks if the provided name is a valid name for a
+// keyspace.
+//
+// As of v17, "all invalid characters" is just the forward slash ("/").
```

Comment:
We should update this to v16.0.2, unless we want to make an exception for this one and get it into v16.0.1 (in code freeze now)?

Reply:
@frouioui and i were planning to get this into v16.0.1. will update

Reply:
Oh yeah, let's change this to `v16.0.1`